[Bug] [kingbase] ERROR: syntax error at or near "t1"

### Search before asking

- [X] I had searched in the [issues](https://github.com/DTStack/chunjun/issues) and found no similar issues.


### What happened

KingbaseES V008R006C008B0020 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-28), 64-bit 该版本执行插入带表别名的sql时报错。


```
Caused by: com.kingbase8.util.KSQLException: ERROR: syntax error at or near "t1";
  Position: 48
	at com.kingbase8.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2575)
	at com.kingbase8.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2311)
	at com.kingbase8.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:319)
	at com.kingbase8.jdbc.KbStatement.executeInternal(KbStatement.java:472)
	at com.kingbase8.jdbc.KbStatement.execute(KbStatement.java:392)
	at com.kingbase8.jdbc.KbPreparedStatement.executeWithFlags(KbPreparedStatement.java:178)
	at com.kingbase8.jdbc.KbPreparedStatement.execute(KbPreparedStatement.java:167)
	at com.dtstack.chunjun.connector.jdbc.statement.FieldNamedPreparedStatementImpl.execute(FieldNamedPreparedStatementImpl.java:173)
	at com.dtstack.chunjun.connector.jdbc.sink.wrapper.SimpleStatementWrapper.writeSingleRecord(SimpleStatementWrapper.java:54)
	at com.dtstack.chunjun.connector.jdbc.sink.wrapper.SimpleStatementWrapper.writeSingleRecord(SimpleStatementWrapper.java:29)
	at com.dtstack.chunjun.connector.jdbc.sink.JdbcOutputFormat.writeSingleRecordInternal(JdbcOutputFormat.java:217)
```

看源码是生成插入sql时会定义一个 t1的别名 

```
INSERT INTO "middle_dev"."dc_table_constraint" t1 ("id", "only_code", "created_time") VALUES (361, '2023-03-22_tableUseTimeTask', '2023-03-23 10:30:14+08');
```


![47289ce4c495c04b410717ce16857c8](https://github.com/user-attachments/assets/6aaceccb-4fe0-4169-98b3-47f91c5b6241)


### What you expected to happen

--

### How to reproduce


```

CREATE TABLE source
(
  `id` int,
  `only_code` string,
  `created_time` timestamp(0)
) WITH (
      'password' = 'xxx',
      'connector' = 'mysql-x',
      'scan.partition.column' = '',
      'scan.parallelism' = '1',
      'scan.where' = '',
      'scan.query-timeout' = '600',
      'table-name' = 'dc_table_constraint',
      'url' = 'jdbc:mysql://xxx:20851/test?useUnicode=true&characterEncoding=utf-8&useSSL=false&connectTimeout=3000&useunicode=true&characterEncoding=utf8&useSSL=false&useCursorFetch=true&tinyInt1isBit=false&zeroDateTimeBehavior=convertToNull',
      'username' = 'xxxx'
      );
CREATE TABLE sink
(
  `id` int,
  `only_code` string,
  `created_time` timestamp(0)
) WITH (
      'schema' = 'middle_dev',
      'password' = 'xxx',
      'connector' = 'kingbase-x',
      'sink.buffer-flush.interval' = '1000',
      'sink.buffer-flush.max-rows' = '1024',
      'sink.all-replace' = 'false',
      'table-name' = 'dc_table_constraint',
      'url' = 'jdbc:kingbase8://xxxx:4321/middle_dev',
      'username' = 'xxxx'
      );
insert into sink 
select 
`id` as `id`,
`only_code` as `only_code`,
`created_time` as `created_time`
 from source ;

```

### Anything else

_No response_

### Version

master

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/chunjun-connectors/chunjun-connector-kingbase/src/main/java/com/dtstack/chunjun/connector/kingbase/dialect/KingbaseDialect.java
+++ b/chunjun-connectors/chunjun-connector-kingbase/src/main/java/com/dtstack/chunjun/connector/kingbase/dialect/KingbaseDialect.java
@@ -118,7 +118,7 @@ public class KingbaseDialect implements JdbcDialect {
                 Arrays.stream(fieldNames).map(f -> ":" + f).collect(Collectors.joining(", "));
         return "INSERT INTO "
                 + buildTableInfoWithSchema(schema, tableName)
-                + " t1 "
+                + " as t1 "
                 + "("
                 + columns
                 + ")"

